ow_lander/scripts/arm_action_servers.py

```diff
@@ -207,6 +211,7 @@ def on_stop_action(self, goal):
         trajectory_async_executer.stop()
 
         rospy.loginfo('%s: Succeeded' % self._action_name)
```

Comment:
I would think that if an action is stopped, it does not succeed.  Thoughts?

Reply:
This is the stop action succeeding

Reply:
Oops, my bad :)